interface/Gruntfile.js

```diff
@@ -96,6 +98,37 @@ module.exports = function ( grunt ) {
     ],
 
     /**
+     * Browserify the ttest library since it's written just for node
+     * Note: this must be run before the copy step (see `build` below).
+     */
+    browserify: {
+      dist: {
+        src: 'node_modules/ttest/hypothesis.js',
+        dest: 'node_modules/ttest/ttest.js',
```

Comment:
- [ ] rename the build product from `ttest.js` to `greenelab.stats.ttest.js` to avoid any future name conflicts.